homeassistant/components/august/__init__.py

```diff
@@ -456,8 +463,33 @@ def _update_locks_detail(self):
 
     def lock(self, device_id):
         """Lock the device."""
-        return self._api.lock(self._access_token, device_id)
+        return _call_api_operation_that_requires_bridge(
+            self.get_lock_name(device_id),
+            "lock",
+            self._api.lock,
+            self._access_token,
+            device_id,
+        )
 
     def unlock(self, device_id):
         """Unlock the device."""
-        return self._api.unlock(self._access_token, device_id)
+        return _call_api_operation_that_requires_bridge(
+            self.get_lock_name(device_id),
+            "unlock",
+            self._api.unlock,
+            self._access_token,
+            device_id,
+        )
+
+
+def _call_api_operation_that_requires_bridge(
+    device_name, operation_name, func, *args, **kwargs
+):
+    """Call an API that requires the bridge to be online."""
+    ret = None
+    try:
+        ret = func(*args, **kwargs)
+    except AugustApiHTTPError as err:
+        raise HomeAssistantError(device_name + ": " + str(err)) from AugustApiHTTPError
```

Comment:
`from AugustAPIHTTPError` is the default in Python 3.

Reply:
Nice, some time saved next time.  Removed the from